packages/terra-hookshot/tests/jest/HookshotDefaultExample.jsx

```diff
@@ -39,9 +39,8 @@ class HookshotDefaultExample extends React.Component {
           type="button"
           id="hookshot-standard-button"
           onClick={this.handleButtonClick}
-        >
-          {''}
-        </button>
+          aria-label=" "
```

Comment:
For these lint failures we will either want to add child text to the button or a valid aria label. Blank aria labels should be avoided, even in tests.